lab_2_keywords_cooccurrence/main.py

```diff
@@ -29,7 +41,34 @@ def extract_candidate_keyword_phrases(phrases: Sequence[str], stop_words: Sequen
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(phrases, list) or not isinstance(stop_words, list) or not stop_words or not phrases:
+        return None
+
+    phrases = [i.lower() for i in phrases]
+    tokens = []
+    keywords = []
+    for sentence in phrases:
+        sublist = []
+        element = ""
+        for symbol in sentence:
+            element += symbol
+        words = element.split()
+        tokens.append(words)
+        for word in words:
+            if word not in stop_words:
+                sublist.append(word)
+            else:
+                if sublist:
+                    sublist = tuple(sublist)
+                    keywords.append(sublist)
+                    sublist = []
+                else:
+                    continue
+        sublist = tuple(sublist)
```

Comment:
you need to do it only if `sublist` is not empty

Reply:
fixed (I guess)